apps/storybook/src/TiledHeatmapMesh.stories.tsx

```diff
@@ -152,11 +172,7 @@ const Template: Story<TiledHeatmapStoryProps> = (args) => {
       <Zoom />
       <SelectToZoom keepRatio modifierKey="Control" />
       <ResetZoomButton />
-      <group
-        scale={[abscissaConfig.flip ? -1 : 1, ordinateConfig.flip ? -1 : 1, 1]}
-      >
```

Comment:
The `flip` is now handled at the layer level to avoid flipping the `TiledTooltipMesh`